dist/atheme.conf.example

```diff
@@ -2206,6 +2206,14 @@ general {
 	 */
 	immune_level = immune;
 
+	/* (*)send_sasl_quit
+	 * If enabled, SaslServ will send a /QUIT before the the services server
+	 * either restarts or shuts down.  Some IRCds unfortunately need this in
+	 * order for cap-notify and the "sasl" cap to function properly.
```

Comment:
Which ircds are those? Doesn't the services server disconnecting already imply a /quit for all its pseudo-clients?